### PR TITLE
Even more rest api

### PR DESCRIPTION
**Pull request by padams**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Tested the changes
- [x] Build (`/path/to/php cli.php cmd=build`) was run locally and any changes were pushed

Re-built owa.reporting package.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

- Added module name to REST urls so that routes are not global to all modules that might add them.
- Added a `/reports` route that will return a `paginatedResultSet`. Will replace use of old style `getResultSet` API method.
- Modified templates to use new `/reports` endpoint instead of old API getResultSet method. Other uses of the old API can be phased out once we convert remaining server rendered UI elements (clickstream, latest visits, etc.) to client side rendered.

## Does this introduce a breaking change?

- [x] Yes
- [ ] No

Only affects the one published route.

## Docs need to be updated?

- [x] Yes
- [ ] No

see: https://github.com/Open-Web-Analytics/Open-Web-Analytics/wiki/REST-API

### CODE REVIEW

**Comment by padams** [comment]
ugh. i just found a bug. hold off for reviewing this.

**Comment by padams** [comment]
ok fixed. review away. ;)

**Comment by Maaiins** [comment]
Will do this, but take a while i need some spare time for this.

File: modules/base/reportVisitors.php [hunk 1]
```diff
@@ -37,7 +37,11 @@ function action() {
 
         $rs = owa_coreAPI::executeApiCommand(array(
             
-            'do'                => 'getLatestVisits',
+            'request_method'	=> 'GET',
```
Comment (Maaiins) [inline modules/base/reportVisitors.php:40]: This is awesome using the same syntax in js an php. Nice work Peter

**Pull request merged**